src/work/ConditionalWork.cpp

```diff
@@ -122,7 +122,8 @@ ConditionalWork::getStatus() const
     {
         elapsed = mStartTimer->checkElapsedTime();
     }
-    return fmt::format("ConditionalWork - Waiting on {} for {} milliseconds.",
-                       getName(), elapsed.count());
+
+    return fmt::format("ConditionalWork - Waiting on {} for {:.1f} seconds.",
```

Comment:
I think we should just replace the `getStatus` here with something much simpler (we don't actually care for how long it's been waiting, it's more important to tell what we're going to do:
```
    return fmt::format("{}{}", mWorkStarted?"":"Waiting before starting ",
                           mConditionedWork->getStatus());
```

Reply:
I think there's still value in logging the time elapsed in case the condition takes longer than expected. I can remove it if you disagree. I've also updated the message to include the status of `mConditionedWork` while waiting.

Reply:
this information would be used while debugging and can be derived from logs @sisuresh

Reply:
Removed elapsed time